utils/i18n.ts

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+import plTranslation from './translations/pl';
+
+// (tip move them in a JSON file and import them)
```

Comment:
I would remove those generic comments :smile:

Reply:
Right, I didn't see it :smile: